src/simulation/Simulation.cpp

```diff
@@ -2833,8 +2835,18 @@ int Simulation::try_move(int i, int x, int y, int nx, int ny)
 		}
 		break;
 	case PT_CNCT:
-		if (y < ny && (TYP(pmap[y+1][x]) == PT_CNCT || TYP(pmap[y+1][x]) == PT_ROCK)) //check below CNCT for another CNCT or ROCK
-			return 0;
+		{
+			float offsetX, offsetY; // Calculate offset from gravity
+			GetGravityField(x, y, 1.0f, 1.0f, offsetX, offsetY);
+			if (offsetX > 0.0f) offsetX = 1.0f;
+			else if (offsetX < 0.0f) offsetX = -1.0f;
+			if (offsetY > 0.0f) offsetY = 1.0f;
+			else if (offsetY < 0.0f) offsetY = -1.0f;
+			if (((nx - x) * offsetX > 0 || offsetX == 0.0f) && // Check direction based on gravity
```

Comment:
I'm not sure what happens here. For e.g. `offsetX = 0.f` and `offsetY = 0.f`, you get a CNCT looking at itself. I'm not sure this is what you want.

Reply:
Fixed (No idea what happened there)